packages/datasets/src/captcha/merkle.ts

```diff
@@ -49,15 +50,15 @@ export class CaptchaMerkleTree {
             layerZero.push(node.hash)
         }
         this.layers.push(layerZero)
-        this.root = this.buildMerkleTree(this.leaves)
+        this.root = this.buildMerkleTree(this.leaves)[0]
```

Comment:
@forgetso can you double check this fn to make sure it's okay?

Reply:
LGTM